habitat-lab/habitat/tasks/rearrange/sub_tasks/pick_task.py

```diff
@@ -61,14 +65,20 @@ def _gen_start_pos(self, sim, episode, sel_idx):
         target_positions = self._get_targ_pos(sim)
         targ_pos = target_positions[sel_idx]
 
-        start_pos, angle_to_obj, was_fail = place_agent_at_dist_from_pos(
-            targ_pos,
-            self._base_angle_noise,
-            self._spawn_max_dist_to_obj,
-            sim,
-            self._num_spawn_attempts,
-            self._filter_colliding_states,
-        )
+        was_fail = True
```

Comment:
Nit: I would change this to `success = False` for clarity.

Reply:
Thank you, Michael! I will prefer `was_fail = True` since I feel that it is more natural to think about that we need to call `place_agent_at_dist_from_pos` if `was_fail = True`